processor/error/package_tests/json_schema_test.go

```diff
@@ -27,6 +27,7 @@ func TestPayloadAttributesInSchema(t *testing.T) {
 		"errors.context.request.cookies.c1",
 		"errors.context.request.cookies.c2",
 		"errors.context.tags.organization_uuid",
+		"errors.context.system.ip",
```

Comment:
This shouldn't be necessary - if so it would be a bug because it actually meant that it is not added to the `fields.yml` (where it is).

Reply:
good catch, thanks! 
fixed